polygon/sync/heimdall_test.go

```diff
@@ -0,0 +1,248 @@
+package sync
+
+import (
+	"context"
+	"github.com/golang/mock/gomock"
+	heimdallclient "github.com/ledgerwatch/erigon/consensus/bor/heimdall"
+	"github.com/ledgerwatch/erigon/consensus/bor/heimdall/checkpoint"
+	"github.com/ledgerwatch/erigon/consensus/bor/heimdall/milestone"
+	heimdallmock "github.com/ledgerwatch/erigon/consensus/bor/heimdall/mock"
+	"github.com/ledgerwatch/log/v3"
+	"github.com/stretchr/testify/assert"
+	"math/big"
+	"testing"
+	"time"
+)
+import "github.com/stretchr/testify/require"
```

Comment:
can move this in the import group above

Reply:
yes, I'll fix in my next PR, not sure why GoLand did it like so, I'll adjust the settings